tartufo/config.py

```diff
@@ -22,11 +23,11 @@
 import truffleHogRegexes.regexChecks
 
 from tartufo import types, util
-
+from tartufo.types import Rule
 
 OptionTypes = Union[str, int, bool, None, TextIO, Tuple[TextIO, ...]]
 
-DEFAULT_REGEXES = truffleHogRegexes.regexChecks.regexes
+DEFAULT_REGEXES = util.convert_regexes_to_rules(truffleHogRegexes.regexChecks.regexes)
```

Comment:
In general, I agree. I try to avoid initializing code at the module level (other than constants), however this was already being done by the trufflehog regex module itself and rather than making too extreme changes and the little risk, this makes sense until a refactor is done. https://github.com/dxa4481/truffleHogRegexes/blob/master/truffleHogRegexes/regexChecks.py#L5

Reply:
Based on meeting conversations I'm thinking that #5 needs to be addressed as part of this.

Reply:
I think it's best to keep the work for the 2 separate, to keep the PRs smaller and move focused. But I do agree they should be done alongside each other. It's really just the order in which they should be tackled that's in question. Those two issues are the only things currently tagged for v2.1 though, so I'm not sure the order matters all that much. 😄

Reply:
#5 is totally unrelated to this change and I agree should not be part of this. Keep things small.